changelog/unreleased/kong/remove_eol_debian_rhel.yml

```
@@ -0,0 +1,2 @@
+message: Debian 10, CentOS 7, and RHEL 7 reached their End of Life (EOL) dates on June 30, 2024. As of this patch, Kong is not building Kong Gateway 3.8.x installation packages or Docker images for these operating systems. Kong is no longer providing official support for any Kong version running on these systems.
```

```suggestion
message: Debian 10, CentOS 7, and RHEL 7 reached their End of Life (EOL) dates on June 30, 2024. As of version 3.8.0.0 onward, Kong is not building any installation packages or Docker images for these operating systems. Kong is no longer providing official support for any Kong version running on these systems.
```
Rephrasing specifically for 3.8.x, since the version isn't out and "patch" isn't technically correct.

---

message has been updated, thanks.